src/Conf.ml

```diff
@@ -295,6 +296,24 @@ module Formatting = struct
     C.choice ~names ~all ~env ~doc ~allow_inline:true ~update:(fun conf x ->
         {conf with infix_precedence= x} )
 
+  let let_open =
+    let doc =
+      "Style of module open. Can be set in a config file with an `let_open \
+       {auto,short,preserve,long}` line. `long` means we use the `let open \
+       Module in (...)` style. `short`means we use `Module.(...)` style. \
+       `auto` means we use the one fitting best. `preserve` keeps the \
+       original style."
+    in
+    let all =
+      [ ("auto", `Auto)
+      ; ("short", `Short)
+      ; ("long", `Long)
+      ; ("preserve", `Preserve) ]
```

Comment:
I am considering setting this default to preserve. It won't churn existing code much, and seems like probably a better default. What do others think?

Reply:
Ok, could you reorder the values to make preserve the default